SetupWifi.md

```diff
@@ -0,0 +1,59 @@
+# Configuration
+
+To connect to a Wifi access point, we need to provide the `wpa_supplicant` with the SSID and passphrase using the command below:
+
+    mkdir /etc/wpa_supplicant
+    wpa_passphrase {YOUR-SSID} {YOUR-PASSPHRASE} > /etc/wpa_supplicant/wpa_supplicant-wlan0.conf
+    
+- In case you want to scan for every SSID in your environment, you can use this command:
+
+      iw wlan0 scan | egrep -w "signal:|SSID:" | paste -d ' ' - - | sort
+      
+  Here is the example output:
+  
+        signal: -42.00 dBm      SSID: MistyWest
+        signal: -43.00 dBm      SSID: MistyDeploy
+        signal: -43.00 dBm      SSID: MistyGuest
+        signal: -60.00 dBm      SSID: TELUS1924
+        signal: -64.00 dBm      SSID: TELUSE55B
+        signal: -70.00 dBm      SSID: TELUSB840
+        signal: -71.00 dBm      SSID: SHAW-714340
+
```

Comment:
move this above the update of the wpa_supplicant file. I think the first thing someone wants to do, most likely is scanning for the SSIDs and then connect to one that's visible, right?